airbyte-integrations/connectors/destination-bigquery/README.md

```diff
@@ -1,3 +1,13 @@
+## Uploading options
+There are 2 available options to upload data to bigquery `Standard` and `GCS Staging`.
+- `Standard` is option to upload data directly from your source to BigQuery storage. This way is faster and requires less resources than GCS one.
```

Comment:
I think we need to explain further about when to choose which option. Its not clear to me when should I choose `Standard` vs `GCS Uploading (CSV format)`

Reply:
Updated, thanks